wikipedia-dark.user.css

```diff
@@ -384,11 +413,297 @@ regexp("https?:\/\/.*wik(i|t).*(org|jp).*") {
   .ve-ui-mwAceEditorWidget .ace_editor {
     border-color: var(--cm-6);
   }
+	/*Blyad's Fixes*/
+	.central-featured-lang :hover, .central-featured-lang :hover strong::before, .other-project-link:hover, .central-featured-lang :hover small::before, .central-featured-lang :hover bdi::before, .central-featured-lang :hover span::before {
+		background-color: #0000;
+		position: relative;
+	}
+	.central-featured-lang :hover::before, .other-project-link:hover::before {
+		content: '';
+		position: absolute;
+		width: 100%;
+		height: 100%;
+		left: 0;
+		top: 0;
+		margin: 0;
+		background-color: var(--blue-3);
+		opacity: .54;
+	}
+	.other-project-link div, .central-featured-lang small, .central-featured-lang strong {
+		filter: brightness(1);
+	}
+	.footer-sidebar-icon.sprite.svg-wikipedia_app_tile {
+		filter: invert(.835) hue-rotate(180deg);
+	}
+	span.jsl10n.sprite.svg-badge_ios_app_store, span.jsl10n.sprite.svg-badge_google_play_store, span.jsl10n.sprite.svg-badge_google_play_store:hover {
+		color: transparent;
+	}
+	[src="//upload.wikimedia.org/wikipedia/donate/a/ac/Close_oojs.png"], [src="//upload.wikimedia.org/wikipedia/commons/thumb/a/a1/OOjs_UI_icon_caretDown.svg/20px-OOjs_UI_icon_caretDown.svg.png"], [src="//upload.wikimedia.org/wikipedia/commons/thumb/d/d8/Stylus-logo.svg/120px-Stylus-logo.svg.png"], [src="//upload.wikimedia.org/wikipedia/commons/thumb/4/4a/Dragon-149393.svg/20px-Dragon-149393.svg.png"] {
```

Comment:
Please use short form

`[src="//upload.wikimedia.org/wikipedia/donate/a/ac/Close_oojs.png"]`

to

`[src*="Close_oojs.png"]`


and so on.

Reply:
Short form in src can overwrite some other unwanted stuff, f.e [here](https://github.com/StylishThemes/Wikipedia-Dark/pull/190#pullrequestreview-797385979) it overwrited the USSR star(or what the hell is this)